src/components/app.js

```diff
@@ -3,13 +3,35 @@ import '../styles/app.scss';
 import dataList from '../data';
 import ContactList from './ContactList';
 
-function App() {
-  return (
-    <div className="view_wrap">
-      <h1 className="view_title">Contact List</h1>
-      <ContactList list={dataList} />
-    </div>
-  );
+class App extends React.Component {
```

Comment:
You can use destructuring in `import React, { Component } from 'react'`

Reply:
Done.